core/src/env/test_env.rs

```diff
@@ -263,6 +263,11 @@ impl TestEnvData {
     pub fn println(&self, content: &str) {
         println!("{}", content)
     }
+
+    /// Deposits an event
+    pub fn deposit_event(&self, data: Vec<u8>) {
+        println!("{:?}", data)
```

Comment:
For testing purposes we definitely are not interested in printing out received events but more into storing them in a vector so that after an execution we can check if we received the correct events. For this you want to add a new field into the `TestEnv` data struct and provide it with getters to the vec (that returns a slice).

Reply:
I have done this I think, but did not introduce a getter. Where would that live?